### PR TITLE
Partial invoices

### PR DESCRIPTION
**Pull request by jumeyermattosarruda**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [x] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

We have updated the architecture of how our OMS communicates with VTEX's Payment gateway so that partial invoices also trigger partial settlements in transactions. The following endpoints had their descriptions changed, to fit the new model:

[Register Change on Order](https://developers.vtex.com/vtex-rest-api/reference/registerchange)
[Invoice notification](https://developers.vtex.com/vtex-rest-api/reference/invoicenotification)
[Retrieve Payment transaction](https://developers.vtex.com/vtex-rest-api/reference/getpaymenttransaction)
[Update order's partial invoice (send tracking number)](https://developers.vtex.com/vtex-rest-api/reference/updatepartialinvoicesendtrackingnumber)

I also used the same PR to fix minor details on the Orders API, sent in through the following feedbacks:
- [EDU-5383](https://vtex-dev.atlassian.net/browse/EDU-5383)
- [EDU-2120](https://vtex-dev.atlassian.net/browse/EDU-2120)
- [EDU-6693](https://vtex-dev.atlassian.net/browse/EDU-6693)
- [EDU-6722](https://vtex-dev.atlassian.net/browse/EDU-6722)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [x] Yes, I already created a release note about this change.
- [ ] No, but I am going to.

### CODE REVIEW

File: VTEX - Orders API.json [hunk 1]
```diff
@@ -8515,7 +8517,8 @@
                         "type": "string"
                     },
                     "statusDescription": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "Status description which is displayed on the Admin panel. This field is obsolete and may not return any value."
```
Comment (karenkrieger) [inline VTEX - Orders API.json:8521]: Eu colocaria um `deprecated` aqui também pra garantir

File: VTEX - Orders API.json [hunk 2]
```diff
@@ -11596,7 +11600,8 @@
...
```
Comment (karenkrieger) [inline VTEX - Orders API.json:11604]: Eu colocaria um `deprecated` aqui também pra garantir

File: VTEX - Orders API.json [hunk 3]
```diff
@@ -2417,7 +2417,7 @@
...
+                "description": "This request allows [changing an order](https://help.vtex.com/en/tutorial/alteracao-de-itens-de-um-pedido-finalizado--tutorials_190) by:\n\r- Adding items\n\r- Removing items\n\r- Applying discount to the total value of the order\n\r- Incrementing the total value of the order. \n\nIn those scenarios of order changes, it is possible to insert a [Partial invoice](https://help.vtex.com/en/tracks/pedidos--2xkTisx4SXOWXQel8Jg8sa/q9GPspTb9cHlMeAZfdEUe). The total value of the order will be updated after the insertion of the invoice, even when there's a partial invoice scenario. The updated value is settled by VTEX's Payment Gateway. The reimbursement for the shopper is automatic. \n\r\n\rThis action can only be done for orders in these status:\n\r- `handling`\n\r- `waiting-for-fulfillment`. \n\r\n\r> The `Change order` resource is needed to use this API request. This is included in `OMS - Full access` and `IntegrationProfile - Fulfillment Oms`, among other default roles available in the Admin. Learn more about the [License manager roles and resources](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#)",
```
[the remark this answers is not in the document]
Comment (jumeyermattosarruda) [reply VTEX - Orders API.json:2420]: É que tô esperando a tradução ficar pronta, mas vai casar com a data de publicação desse PR

File: VTEX - Orders API.json [hunk 4]
```diff
@@ -12039,7 +12044,8 @@
                         "type": "string"
                     },
                     "statusDescription": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "Status description which is displayed on the Admin panel. This field is obsolete and may not return any value."
```
[the remark this answers is not in the document]
Comment (jumeyermattosarruda) [reply VTEX - Orders API.json:12048]: Added!

**Comment by jumeyermattosarruda** [comment]
O Spectral ainda aponta erros que já foram corrigidos e commitados. É estranho também que os erros apontados pra mim ao _Re-run all jobs_ são diferentes dos erros que aparecem para a Karen. De qualquer forma, todos os erros já foram corrigidos, o Spectral que não parece captar isso /:

File: VTEX - Orders API.json (hunk 3 above)
Comment (karenkrieger) [inline VTEX - Orders API.json:2446]:
```suggestion
                "description": "This request allows [changing an order](https://help.vtex.com/en/tutorial/alteracao-de-itens-de-um-pedido-finalizado--tutorials_190) by:\n\r- Adding items\n\r- Removing items\n\r- Applying discount to the total value of the order\n\r- Incrementing the total value of the order. \n\nIn those scenarios of order changes, it is possible to insert a [Partial invoice](https://help.vtex.com/en/tracks/pedidos--2xkTisx4SXOWXQel8Jg8sa/q9GPspTb9cHlMeAZfdEUe). The total value of the order will be updated after the insertion of the invoice, even when there's a partial invoice scenario. The updated value is settled by VTEX's Payment Gateway. The reimbursement for the shopper is automatic. \n\r\n\rThis action can only be done for orders in these status:\n\r- `handling`\n\r- `waiting-for-fulfillment`. \n\r\n\r> The `Change order` resource is needed to use this API request. This is included in `OMS - Full access` and `IntegrationProfile - Fulfillment Oms`, among other default roles available in the Admin. Learn more about the [License manager roles and resources](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#).",
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**